### DISCUSSION TITLE
Add Logout Function in my Custom Account Controller

### DISCUSSION THREAD

**Discussion by YoumnaJ** [OrchardCMS/OrchardCore#14927] [Q&A]
hello I need some help please
I Add Logout function to my Custom Account controller and I put this Route in my module startup 

 routes.MapAreaControllerRoute(
     name: "LogOff",
     areaName: "Test.OrcharCore.User",
     pattern: "Account/LogOff",
     defaults: new { controller = "Account", action = "LogOff" }
 );

And I add in my zone template this code
<form asp-controller="Account" asp-action="LogOff" method="post" class="no-multisubmit">
        <button type="submit" class="dropdown-item">
            <i class="fa-solid fa-sign-out-alt" aria-hidden="true"></i> Logout
        </button>
</form>
but the problem is that when clicking on logout nothing happen and doesn't enter into the custom logoff function created in AccountController

Thank you ,

**Comment by weirdyang** [comment]
the areaName should correspond to your feature name, check if your feature name is "test"?
